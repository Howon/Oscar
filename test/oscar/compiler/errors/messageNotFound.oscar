message testNow()


actor Tester() {

  receive = {
    | testNow() => {
        Println("Error");
      }
    | die() => {
      }
  }
}

def main() => unit = {
  actor<Tester> tester = spawn actor<Tester>();
  message<testLater>() |> tester;
  message<die>() |> tester;
}
