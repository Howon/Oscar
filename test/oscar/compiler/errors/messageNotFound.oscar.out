Message of type testLater not found